x/authz/keeper/keeper_test.go

```diff
@@ -196,6 +196,55 @@ func (s *TestSuite) TestKeeperFees() {
 	s.Require().NotNil(authorization)
 }
 
+// Tests that all msg events included in an authz MsgExec tx
+// Ref: https://github.com/cosmos/cosmos-sdk/issues/9501
+func (s *TestSuite) TestDispatchedEvents() {
+	app, addrs := s.app, s.addrs
+	granterAddr := addrs[0]
+	granteeAddr := addrs[1]
+	recipientAddr := addrs[2]
+	s.Require().NoError(simapp.FundAccount(app.BankKeeper, s.ctx, granterAddr, sdk.NewCoins(sdk.NewInt64Coin("steak", 10000))))
+	now := s.ctx.BlockHeader().Time
+	s.Require().NotNil(now)
```

Comment:
if we do:
```
require := s.Require()
```
then we can use eg: `require.NotNil(...)` and the code is easier to read.